src/frontend/packages/core/package.json

```diff
@@ -30,9 +30,11 @@
     "extract-translations": "formatjs extract './**/*.ts*'  --ignore ./node_modules --ignore './**/*.d.ts' --ignore ./tsconfig.tsbuildinfo --ignore './**/*.stories.ts*' --out-file './i18n/frontend.json' --id-interpolation-pattern '[sha512:contenthash:base64:6]' --format crowdin",
     "lint": "eslint 'src/**/*.{ts,tsx}'",
     "format:check": "prettier --check src",
-    "format:write": "prettier --write src"
+    "format:write": "prettier --write src",
+    "build-theme": "cunningham -g css -o src"
   },
   "dependencies": {
+    "@openfun/cunningham-react": "0.7.0",
```

Comment:
During `yarn install` there are several warnings. In order to fix most of them, you should add `react` and `react-dom` as dev dependencies.